### PR TITLE
[stable/parse] Fix ingress setup and update MongoDB dependency

### PR DESCRIPTION
**Pull request by juan131**
#### What this PR does / why we need it:

This PR fixes the ingress setup by allowing to configure Parse Dashboard so it redirects the requests to Parse Server through the Ingress controller.

On the other hand, it bumps the major version of MongoDB chart (dependency).

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/parse/templates/NOTES.txt [hunk 1]
```diff
@@ -62,14 +68,18 @@ service:
...
 
-{{ else }}
-1. Get the Parse Dashboard URL by running:
+{{- else }}
+
+1. Get the Parse Dashboard URL:
+
+{{- if .Values.ingress.enabled }}
+
+  You should be able to access Parse Dashboard trough: {{- range .Values.ingress.dashboard.hosts }} {{ if .tls }}https{{ else }}http{{ end }}://{{ .name }} {{- end }}
```
Comment (alemorcuq) [inline stable/parse/templates/NOTES.txt]: through*

File: stable/parse/templates/NOTES.txt [hunk 2]
```diff
@@ -9,24 +8,31 @@ Parse Server
 
 1. Get your Parse Server URL:
 
-{{- if contains "NodePort" .Values.service.type }}
+{{- if .Values.ingress.enabled }}
+{{ $ingressHost := first .Values.ingress.server.hosts }}
+
+  You should be able to access Server trough: {{- range .Values.ingress.server.hosts }} {{ if .tls }}https{{ else }}http{{ end }}://{{ .name }} {{- end }}
```
Comment (alemorcuq) [inline stable/parse/templates/NOTES.txt]: You should be able to access Parse Server through*

**Comment by alemorcuq** [comment]
Two minor things in the Notes.txt

**Review by alemorcuq — APPROVED** [review]
lgtm

**Comment by juan131** [comment]
Closing PR in favor of https://github.com/helm/charts/pull/16407

**Pull request closed without merging**